pages/key-manager/quickstart.mdx

```diff
@@ -16,6 +16,11 @@ Scaleway's Key Manager allows you to create key encryption keys from the [Scalew
 - A Scaleway account logged into the [console](https://console.scaleway.com)
 - [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
 
+## Console overview
+
+Discover the Key Manager interface on the Scaleway console.
+<GuideFlow src="https://app.guideflow.com/embed/9r25wn4tzr"/>
```

Comment:
Just a couple of nitpicky comments:

- This isn't about the Guideflow, but it made me realise we're still not consistent with whether we capitalize "region" or not:
<img width="1545" height="402" alt="image" src="https://github.com/user-attachments/assets/dc169038-5da3-4025-b9ee-9063ae2e078d" />

- I think in this step, you should probable delete the bit about the Overview page displaying, because it is repeated in the next step
<img width="861" height="513" alt="image" src="https://github.com/user-attachments/assets/f3a39c22-ef11-4562-93e9-6bd8257df188" />